### DISCUSSION TITLE
怎么链接模组和MC百科

### DISCUSSION THREAD

**Discussion by WarmurCle** [Meloong-Git/PCL#1869] [提问] [answered]
我自己做了个模组，上传到了MC百科和Forge，怎么才可以在PCL的模组下载显示MC百科链接和中文译名啊？

**Comment by WForst-Breeze** [comment, marked as answer]
PCL2 的 MC 百科信息对照表会定期进行同步、更新并储存至 [ModData.txt](https://github.com/Hex-Dragon/PCL2/blob/5c2a78e5e01c6ed3ccf6d4d20d796c583cb61119/Plain%20Craft%20Launcher%202/Resources/ModData.txt#L4) 中。同步完成后您的模组就可以在 PCL2 中正常显示中文译名及百科信息了~
